action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
     description: 'The version of Bundler to install. Either none, 1, 2, latest or Gemfile.lock. The default tries Gemfile.lock and otherwise uses latest.'
     required: false
     default: 'default'
+  bundle-install:
+    description: 'Run "bundle install", and cache the result automatically.'
+    required: false
+    default: 'true'
```

Comment:
I'd like to have some feedback on new inputs for this PR.
I think caching should be enabled by default because it's a major time saver (and reduces load to rubygems.org).

I don't think it's useful to have an input for "bundle install but don't cache" as users can just `- run: bundle install` for that.

I think we should try to cache both when `Gemfile.lock` exists and when it doesn't.
If there is no `Gemfile.lock`, we'll resolve the gems and install newer gems if there is any new release (both `bundle check` and `bundle install` do that if no Gemfile.lock).

Reply:
I've renamed the input to `bundler-cache`